engine/src/main/java/org/destinationsol/game/Hero.java

```diff
@@ -67,18 +67,18 @@ public boolean isNonTranscendent() {
     }
 
     public Pilot getPilot() {
-        return isTranscendent ? transcendentHeroShip.getPilot() : hero.getPilot();
+        return isTranscendent ? transcendentHeroShip.getPilot() : shipHero.getPilot();
     }
 
-    public SolShip getHero() {
+    public SolShip getShipHero() {
```

Comment:
Not sure about `getShipHero`. How does `getShip` sound? Short, clear and to the point.